.gitmodules

```diff
@@ -1,3 +1,9 @@
 [submodule "avrlib"]
 	path = avrlib
-	url = https://github.com/pichenettes/avril
+	url = https://github.com/pichenettes/avril.git
+[submodule "avrlibx"]
+	path = avrlibx
+	url = https://github.com/pichenettes/avrilx.git
+[submodule "avr_audio_bootloader"]
```

Comment:
Do we need these?  Might be better to keep it simple

Reply:
avr_audio_bootloader is needed for compile successful.
But avrlibx isn't needed. So I fixed to remove avrlibx.